src/tests/plugins/adapters/AnalyticsTrackerAdapter.test.js

```diff
@@ -140,6 +140,24 @@ describe('AnalyticsTrackerAdapter', () => {
     expect(analyticsClientSpy.track).toHaveBeenCalledWith('User Failed to Sign In', {})
   })
 
+  it('should that the correct tracker is being used by verifying that the URL and location parameters are passed correctly.', () => {
```

Comment:
não colocar detalhes da implementação no título do teste (used by verifying that the URL and location). Como URL e location são nome de propriedades do seu parâmetro. o melhor é sempre escrever apenas parameters e não o nome das suas propriedades internas ( detalhe de implementação), pois isso pode e normalmente vai mudar com o tempo e teremos que lembrar de atualizar o título do teste.

Removendo detalhe de implementação do título, mostrando que o teste foca que os parâmetros do evento sejam passados corretamente.
should use the create event  with correct parameters.